libexec/tgenv-list-remote

```diff
@@ -9,13 +9,33 @@ if [ ${#} -ne 0 ];then
   exit 1
 fi
 
-link_release="https://api.github.com/repos/gruntwork-io/terragrunt/tags?per_page=1000"
-print=$(curl --tlsv1.2 -sf $link_release)
+GITHUB_API_HEADER_ACCEPT="Accept: application/vnd.github.v3+json"
+
+temp=`basename $0`
+TMPFILE=`mktemp /tmp/${temp}.XXXXXX` || exit 1
+
+function rest_call {
+    curl --tlsv1.2 -sf $1 -H "${GITHUB_API_HEADER_ACCEPT}" | sed -e 's/^\[$//g' -e 's/^\]$/,/g' >> $TMPFILE
+}
+
+# single page result-s (no pagination), have no Link: section, the grep result is empty
+last_page=`curl -I --tlsv1.2 -s "https://api.github.com/repos/gruntwork-io/terragrunt/tags?per_page=100" -H "${GITHUB_API_HEADER_ACCEPT}" | grep '^link:' | sed -e 's/^link:.*page=//g' -e 's/>.*$//g'`
```

Comment:
Just a heads up, I was testing this fix and found that the pagination did not work unless `link` was capitalized.